vulture/core.py

```diff
@@ -255,6 +255,29 @@ def by_size(item):
         return sorted(confidently_unused,
                       key=by_size if sort_by_size else by_name)
 
+    def make_whitelist(self, min_confidence=0, sort_by_size=False):
+        unused_code = self.get_unused_code(
+                min_confidence=min_confidence, sort_by_size=sort_by_size)
+        if any(item.typ in ('attribute', 'property') for item in unused_code):
+            print("""\
+class VultureMock:
+    def __init__(self, _):
```

Comment:
`__init__` must be `__getattr__`. Given that the tests pass nonetheless, we are not testing the right things...

Reply:
Yes, that's a mistake. Although this code is erred, Vulture successfully gets the unused attribute/property into the `used_attr` set and therefore starts whitelisting the results. So, I guess we might be testing the correct thing, am I correct?

Reply:
We might want to check the sanctity of the code by running it with python, but I'm not sure that we would want to do that, should we?

Reply:
The resulting whitelist must be syntactically correct, but it usually can't be executed. Our tests chek the syntax in the scan method, so that's fine. Since the result can't be excuted, we also don't need the VultureMock class after all. We are testing the correct thing after all :-)

Reply:
So, from what I get - we just print `_.foobar  # unused attrivute (...)` without explicitly declaring what is `_` since that's enough to whitelist foobar from being reported by Vulture, right?

Reply:
Right.